src/brevitas/export/onnx/handler.py

```diff
@@ -88,13 +88,13 @@ def __init__(self):
         self.debug_input = False
         self.debug_output = False
 
-    # @abstractmethod
-    # def prepare_for_export(self, module):
-    #     pass
-
-    # @abstractmethod
-    # def symbolic_execution(self, *args, **kwargs):
-    #     pass
+    @abstractmethod
```

Comment:
Is this necessary given that it's already defined in BaseHandler?

Reply:
No